lib/livebook/release.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Livebook.Release do
+  @moduledoc false
+
+  @doc """
+  Runs the setup for all apps deployed from directory on startup.
+  """
+  def run_apps_setup() do
```

Comment:
If you want a different name: `warmup_apps`. A bit more general in case want to do other stuff in the future. Then we would have to rename the scripts too. :)

Reply:
Beautiful, I was looking for a better name!